### ISSUE TITLE
Multiple SPFx apps installed through the private app catalog invisible in site contents

### ISSUE THREAD

**Issue by pedro-pedrosa** [SharePoint/sp-dev-docs#1423]
#### Category
- [ ] Question
- [ ] Typo
- [x] Bug
- [ ] Additional article idea

#### Expected or Desired Behavior

When multiple apps are installed on a site, they should all be visible under site contents in modern experience.

#### Observed Behavior

Only the latest app is displayed in the site contents modern page. If switched to classic view, all apps are visible.

These were my exact steps:
1. Added the first app, which was visible in site contents
2. Added the second app, which was visible but made the first app invisible
3. Added the third app. At this point, I couldn't see either the first or the second app but I was able to see the third app actually listed twice. Even the status styling (ie grayed out while installing/uninstalling) was the same for both
4. Uninstalled the third app. The second app was being listed again only once.
5. Updated the first app. This had no effect on the visibility of the apps

#### Steps to Reproduce

Install at least two apps in a site. I have tested this with three SPFx apps using standard toolchain (yeoman generator, gulp bundle --ship, gulp package-solution --ship) which I uploaded to a private app catalog.

**Comment by ahackettms** [comment]
We are working on a fix for this. Will update once the fix is ready. Thank you

**Comment by ecraig12345** [comment]
This has been fixed internally, and the fix should roll out by July 25.

(@VesaJuvonen can you please mark this issue as fixed?)

**Comment by VesaJuvonen** [comment]
Thanks @ecraig12345, we typically close items only after the fix has been rolled out and that we have confirmed that the fix is working with the people who reported it. This is to ensure that we don't close things, which have not been confirmed. Let's wait until it's actually rolled out so that we truly confirm that fix has been successfully deployed.

**Comment by ecraig12345** [comment]
Sounds good!

**Comment by VesaJuvonen** [comment]
This issues should be fully solved and fixed, given the work we have done in engineering in this area. To ensure that we clean up the backlog of the issues, we will be closing this one. If you still have issues on this particular area, please open up a new issue so that we can get updated issue description and reproduce steps. We do apologize the non-optimal handling of these old issues and are looking into improving this in future.
